README.md

```diff
@@ -2,20 +2,20 @@
 
 [![Gitter](https://badges.gitter.im/hail-is/hail.svg)](https://gitter.im/hail-is/hail?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![CI Status](https://ci.hail.is/app/rest/builds/buildType:(id:HailSourceCode_HailCi)/statusIcon)](https://ci.hail.is/viewType.html?buildTypeId=HailSourceCode_HailCi&branch_HailSourceCode_HailMainline=%3Cdefault%3E&tab=buildTypeStatusDiv)
 
-[Hail](https://hail.is) is an open-source, scalable framework for exploring and analyzing genomic data. Starting from sequencing or microarray data in [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf) and [other formats](https://hail.is/hail/hail.HailContext.html#hail.HailContext), Hail can, for example:
+[Hail](https://hail.is) is an open-source, scalable framework for exploring and analyzing genomic data. The Hail project began in Fall 2015 to empower the worldwide genetics community to [harness the flood of genomes](https://www.broadinstitute.org/blog/harnessing-flood-scaling-data-science-big-genomics-era) to discover the biology of human disease. Hail has been used for dozens of major studies and is the core analysis platform of large-scale genomics efforts such as [gnomAD](http://gnomad.broadinstitute.org/).
 
+Starting from sequencing or microarray data in [VCF](https://samtools.github.io/hts-specs/VCFv4.2.pdf), [BGEN](http://www.well.ox.ac.uk/~gav/bgen_format/bgen_format_v1.2.html) or [PLINK](https://www.cog-genomics.org/plink2/formats) format, Hail can, for example:
+
+ - load variant and sample annotations from text tables, JSON, VCF, VEP, and locus interval files
  - generate variant annotations like call rate, Hardy-Weinberg equilibrium p-value, and population-specific allele count
  - generate sample annotations like mean depth, imputed sex, and TiTv ratio
- - load variant and sample annotations from text tables, JSON, VCF, VEP, and locus interval files
- - generate new annotations from existing annotations and the genotypes, and use these to filter samples, variants, and genotypes
+ - generate new annotations from existing ones and the genotypes, and use these to filter samples, variants, and genotypes
```

Comment:
This line sounds weird to me. Specifically "from existing ones and the genotypes".  Maybe if it said "from **the** existing ones" or something so there would be a "the" before both. 

Or maybe "generate new annotations from existing annotations as well as genotypes,". Idk, this seems nitpicky.